superset-frontend/package.json

```diff
@@ -178,7 +178,7 @@
     "react-lines-ellipsis": "^0.15.0",
     "react-loadable": "^5.5.0",
     "react-query": "^3.39.2",
-    "react-redux": "^7.2.0",
```

Comment:
Why do we upgrade to this particular version? According to react-redux changelog the compatibility with react 18 starts with 8.0.0 (and the latest version is 8.0.5). 8.x also seems to be compatible with older versions of react so maybe we could use that?

Reply:
I tried updating to react-redux to 8.0.5 but there is a type issue with action returned from `useDispatch()`. 
I also tried the recommended solution from https://redux.js.org/usage/usage-with-typescript#define-typed-hooks still  didn't resolve the type issue.  `dispatch<any>()` work but it only silent the error. We can hold off the update until we figure how  to resolve this type issue